resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,152 @@
+"""Converting bam to bed and normalised bigwig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    Process,
+    SchedulingClass,
+    FileField,
+    StringField
+)
+
+
+class BamToBed(Process):
```

Comment:
Maybe this should be called `BamToBedpe`, since the output is a bedpe file. This should also be reflected in the docstring, slug... below.